runtime/compiler/runtime/RelocationRuntime.cpp

```diff
@@ -132,23 +132,24 @@ char *TR_RelocationRuntime::_reloErrorCodeNames[] =
    "methodFromSingleInterfaceImplValidationFailure",   // 41
    "methodFromSingleAbstractImplValidationFailure",    // 42
    "j2iThunkFromMethodValidationFailure",              // 43
-   "svmValidationFailure",                             // 44
-   "wkcValidationFailure",                             // 45
-
-   "classAddressRelocationFailure",                    // 46
-   "inlinedMethodRelocationFailure",                   // 47
-   "symbolFromManagerRelocationFailure",               // 48
-   "thunkRelocationFailure",                           // 49
-   "trampolineRelocationFailure",                      // 50
-   "picTrampolineRelocationFailure",                   // 51
-   "cacheFullRelocationFailure",                       // 52
-   "blockFrequencyRelocationFailure",                  // 53
-   "recompQueuedFlagRelocationFailure",                // 54
-   "debugCounterRelocationFailure",                    // 55
-   "directJNICallRelocationFailure",                   // 56
-   "ramMethodConstRelocationFailure",                  // 57
-
-   "maxRelocationError"                                // 58
+   "isClassVisibleValidationFailure",                  // 44
```

Comment:
You didn't have to reorder all the error codes because of https://github.com/eclipse-openj9/openj9/blob/7d076b59a1f3d912e810effc5bb2c4820c011822/runtime/compiler/runtime/RelocationRuntime.hpp#L136-L140

However, since you've already done the work, it's fine as is :)

Reply:
No action required to resolve :)